src/docs/sphinx/advancedExamples/validationStudies/carbonStorage/isothermalHystInjection/Example.rst

```diff
@@ -0,0 +1,342 @@
+    :language: xml
+    :start-after: <!-- SPHINX_FLUID -->
+    :end-before: <!-- SPHINX_FLUID_END -->
+
+As this benchmark is testing the sensitivity of the plume dynamics to the relative permeability hysteresis modeling,
+in commented block the **TableRelativePermeabilityHysteresis** block sets up bounding curves for imbibition and drainage
+under `imbibitionNonWettingRelPermTableName`, `imbibitionWettingRelPermTableName` and, `drainageWettingNonWettingRelPermTableNames`
+compared to the `wettingNonWettingRelPermTableNames` label of the drainage only **TableRelativePermeability** blocks.
+Those link to **TableFunction** blocks in **Functions**, which define sample points for piecewise linear interpolation.
+This feature is used and explained in more details in the following section dedicated to *Initial and Boundary conditions*.
+
+See,
+
+.. code-block:: console
+
+  ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_hysteresis_iterative_base.xml
+
+for the input base with hysteresis set up.
+
+.. note::
+    `wettingNonWettingRelPermTableNames` in **TableRelativePermeability** and `drainageWettingNonWettingRelPermTableNames` in
+    **TableRelativePermeabilityHysteresis** usage are identical.
+
+
+Capillary pressure is also tabulated and defined in **TableCapillaryPressure**. No hysteresis is modeled yet on the capillary pressure.
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_drainageOnly_iterative_base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_FLUID -->
+    :end-before: <!-- SPHINX_FLUID_END -->
+
+-----------------------------------------------------------
+Initial and boundary conditions
+-----------------------------------------------------------
+
+The domain is initially saturated with brine with a hydrostatic pressure field.
+This is specified using the **HydrostaticEquilibrium** XML tag in the **FieldSpecifications** block.
+The datum pressure and elevation used below are defined in
+`(Class et al., 2009) <https://link.springer.com/article/10.1007/s10596-009-9146-x>`__).
+
+.. literalinclude:: ../../../../../../inputFiles/compositionalMultiphaseFlow/benchmarks/Class09Pb3/class09_pb3_drainageOnly_iterative_base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_HYDROSTATIC -->
+    :end-before: <!-- SPHINX_HYDROSTATIC_END -->
+
+In the **Functions** block, the **TableFunction** s named ``initOilCompFracTable`` and ``initWaterCompFracTable``
+define the brine-saturated initial state, while the **TableFunction** named ``initTempTable`` defines the
+homogeneous temperature field (temperature is not used in this benchmark).   
+		 
+Since the fluid densities are constant for this simple incompressible fluid model, the same result
+could have been achieved using a simpler table in a **FieldSpecification** tag, as explained
+in :ref:`EquilibriumInitialCondition`.
```

Comment:
```suggestion
```
Same here, I would remove because there a mildly nonlinear dependence of brine density on pressure (small) and on temperature (larger), and here we consider a geothermal gradient.

I agree this is slightly inconsistent with the way boundary conditions are imposed...